Dockerfiles/ci/run_tests.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import unittest
+import os
+import os.path
+from os import path
+
+class RunOperatorTestPlaybookTests(unittest.TestCase):
+
+    def setUp(self):
+        self.OPERATOR_DIR = os.getenv('OPERATOR_DIR',
+                                      "/project/operator_dir/")
+        self.OPERATOR_WORK_DIR = os.getenv('OPERATOR_WORK_DIR',
+                                           "/project/test_operator_work_dir/")
+        self.WORK_DIR = os.getenv('WORK_DIR',
+                                  "/project/output/")
+        self.PLAYBOOKS_DIR = os.getenv('PLAYBOOKS_DIR',
```

Comment:
Do not use uppercase names for clamm members

run_tests.py:11:8: C0103: Attribute name "OPERATOR_DIR" doesn't conform to snake_case naming style (invalid-name)
run_tests.py:13:8: C0103: Attribute name "OPERATOR_WORK_DIR" doesn't conform to snake_case naming style (invalid-name)
run_tests.py:15:8: C0103: Attribute name "WORK_DIR" doesn't conform to snake_case naming style (invalid-name)
run_tests.py:17:8: C0103: Attribute name "PLAYBOOKS_DIR" doesn't conform to snake_case naming style (invalid-name)

Reply:
updated